Bundle local static React Native assets

dcalhoun opened the pull request:
* **Gutenberg Mobile:** https://github.com/wordpress-mobile/gutenberg-mobile/pull/3810

Now that we attempt to load local static image assets in the React Native bundle in https://github.com/WordPress/gutenberg/pull/33790, we must copy those assets for them to properly show up in the WordPress iOS app build. 

To test: 
1. Use an app build that does not rely upon a local React Native Metro server. 
2. Open the block editor. 
3. Tap the three-dot menu button in the top-right corner. 
4. Tap Help. 
5. Tap one of the available articles.
6. ℹ️ **Expected:** Verify the images are displayed and not missing. 

<details><summary>Screenshot of Help Image</summary>

<img width="276" alt="Gutenberg Help image" src="https://user-images.githubusercontent.com/438664/128557045-6ecfaf76-dec4-4515-912b-df913b8f54e1.png">

</details>

## Regression Notes
1. Potential unintended areas of impact
  Modifications to the build configuration could cause the build process to fail. 
2. What I did to test those areas of impact (or what existing automated tests I relied on)
  Verified the Debug build succeeded. 
3. What automated tests I added (or what prevented me from doing so)
  I do not believe this could/should be tested with automation, as build failures should be fairly obvious. 

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

dcalhoun committed:
Bundle local static React Native assets

Now that we are attempting to load local static image assets in the
React Native bundle, we must copy those assets for them to properly show
up in the WordPress iOS app.

ceyhun reviewed (commented):
Thanks @dcalhoun! Tested it with the WordPress target with debug config and it's working great! Do you think maybe we should add the same change to Jetpack target as well?
Also maybe we can trigger the installable builds to verify if it builds correctly on CI with release config for both targets.

dcalhoun commented:
> Do you think maybe we should add the same change to Jetpack target as well?

I had not considered that. Based on my understanding of how the Jetpack app also consumes Gutenberg, I would imagine we should update Jetpack's target as well. I will do that. Thank you! 

> Also maybe we can trigger the installable builds to verify if it builds correctly on CI with release config for both targets.

Great idea. I started a CI task now to create one.

dcalhoun committed:
Jetpack bundles local static React Native assets

Now that we are attempting to load local static image assets in the
React Native bundle, we must copy those assets for them to properly show
up in the WordPress iOS app.

dcalhoun commented:
@ceyhun I updated the Jetpack target configuration to match, and verified the images loaded via a build with Xcode. Additionally, I created installable builds for both WordPress and Jetpack; those are attached to this PR. Ready for another review when you have time. Thanks!

ceyhun reviewed (approved):
Verified that both builds have the images. Thanks David!
